std_msgs/Header header 
# List of detected surface mines
minesweepers_gazebo/MineDetection[] surface_mines
# List of detected buried mines
minesweepers_gazebo/MineDetection[] buried_mines
